plugins/webpack/test/tasks/webpack.test.ts

```diff
@@ -45,7 +45,7 @@ describe('webpack', () => {
       expect(fork).toBeCalledWith(
         webpackCLIPath,
         ['build', '--color', '--mode=production', '--config=webpack.config.js'],
-        { silent: true }
+        { silent: true, execArgv: expect.arrayContaining([]) }
```

Comment:
**suggestion**: i think we should add a test for the legacy openssl behaviour, by mocking or overriding `process.allowedNodeEnvironmentFlags` if possible